packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -97,9 +97,10 @@ export class AtomicSearchInterface {
   }
 
   private initSearchParameterManager() {
-    const stateWithoutHashSign = window.location.hash.slice(1);
+    const stateWithoutHash = window.location.hash.slice(1);
+    const decodedState = decodeURIComponent(stateWithoutHash);
```

Comment:
I initially made this change in response to your comment @ThibodeauJF, on KIT-265. I either forgot to push the commit or my push was rejected, so it was not included as part of that PR. It will be in in this time,